dcos/subcommand.py

```diff
@@ -118,6 +124,30 @@ def distributions():
         return []
 
 
+def command_info(command):
+    """top level documentation of default DCOS-CLI command
+    :param command: name of command
+    :param type: str
+    :returns: command summary
+    :rtype: str
+    """
+
+    return command_documentation(command).split('\n')[0]
```

Comment:
Ah Right. I see the problem. Maybe the default subcommand is a map of string to module so we can do something like `default_modules[command].help_doc()` or `default_modules[command].main(options)`.